manifests/guestcluster/1.20/pvcsi.yaml

```diff
@@ -422,6 +422,7 @@ data:
   "online-volume-extend": "true"
   "file-volume": "false"
   "csi-sv-feature-states-replication": "false"
+  "block-volume-snapshot": "false"
```

Comment:
Do we have to update guest and supervisor manifests about the FSS?

Reply:
Yes, I was told to explicitly specify this to indicate that it's not supported in guest and supervisor.